Makefile

```diff
@@ -214,66 +213,46 @@ generate-go-apis: ## Alias for .build/generate-go-apis
 		paths=./bootstrap/eks/api/... \
 		paths=./controlplane/eks/api/... \
 		paths=./iam/api/... \
-		crd:crdVersions=v1 \
 		output:crd:dir=config/crd/bases \
-		output:webhook:dir=$(WEBHOOK_ROOT) \
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+		crd:crdVersions=v1 \
+		rbac:roleName=manager-role \
 		webhook
 
```

Comment:
We seem to have lost the controller-gen on the `controllers` folders...i think this needs to be added back in.

Reply:
Apart from ^^^ that this looks good.

Reply:
Do we need controllers folder in controller-gen? Is there any generated code in that folder? I actually didn't find any.

Reply:
All of the RBAC generation comes from the controllers

Reply:
So its needed :)

Reply:
ok :) Will add it

Reply:
Added, PTAL